docs/journal/00-introducing-the-journey.md

```diff
@@ -1,118 +1,110 @@
 # Introducing the Journey
 
-Smart buildings based on IoT solutions are becoming increasingly commonplace. A smart building tracks and controls the internal environment. In particular thermostatic and photo-electric devices can monitor factors such as the temperature, ambient lighting, and humidity to help ensure the comfort of the occupants. A smart building can also incorporate safety devices, such as smoke detectors and intruder alarms. The data captured from the various monitoring devices can be used to send commands to other devices such as air conditioning units, lighting dimmers, sprinklers, and automatic fire doors, as well as alert the appropriate authorities if an emergency situation is detected.
+This project has two aspects:
+
+- Guidance on designing an IoT solution, with a particular focus on the cloud back end.
+- A reference architecture.
+
+The reference architecture is designed to test and validate our assumptions, and to keep our guidance honest. 
 
 > To find out what we mean by "IoT solution", see [_What is an IoT solution?_][intro-to-iot].
 
-## Meet Fabrikam
+## The problem space  
 
-Fabrikam is a startup software development company with a dozen employees. The developers all have a background with .NET and Microsoft Azure. They also have a growing interest in technologies such as NodeJS, Hadoop, and Java.
+We've identified some common scenarios and challenges in IoT. We're interested to hear from other people about their experiences. If you've worked on an IoT solution, how well does it map to what we are presenting?
```

Comment:
> We're interested to hear from other people about their experiences. If you've worked on an IoT solution, how well does it map to what we are presenting?

At some point, once the content has matured, we will want to remove these kinds of comments. Should we do something to mark as such?